queue/driver_machinery.go

```diff
@@ -0,0 +1,85 @@
+// TODO: Will be removed in v1.17
+
+package queue
+
+import (
+	"time"
+
+	"github.com/RichardKnop/machinery/v2"
+	redisbackend "github.com/RichardKnop/machinery/v2/backends/redis"
+	redisbroker "github.com/RichardKnop/machinery/v2/brokers/redis"
+	"github.com/RichardKnop/machinery/v2/config"
+	"github.com/RichardKnop/machinery/v2/locks/eager"
+	"github.com/RichardKnop/machinery/v2/log"
+
+	logcontract "github.com/goravel/framework/contracts/log"
+	"github.com/goravel/framework/contracts/queue"
+)
+
+type Machinery struct {
+	connection string
+	config     *Config
+	log        logcontract.Log
+}
+
+func NewMachinery(connection string, config *Config, log logcontract.Log) *Machinery {
```

Comment:
Use the `Config` interface:

```suggestion
func NewMachinery(connection string, config queue.Config, log logcontract.Log) *Machinery {
```